plugin/src/styles/output.css

```
@@ -1511,216 +972,52 @@ video {
-  --tw-border-opacity: 1;
-  border-color: rgb(17 24 39 / var(--tw-border-opacity));
-}
-
-.border-green-800 {
-  --tw-border-opacity: 1;
-  border-color: rgb(22 101 52 / var(--tw-border-opacity));
-}
-
-.border-primary-bg {
-  border-color: var(--figma-color-bg);
+  border-color: rgb(209 213 219 / var(--tw-border-opacity, 1));
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)